src/paxcounter_orig.conf

```diff
@@ -16,7 +16,7 @@
 
 // Set this to include BLE counting and vendor filter functions, or to switch off WIFI counting
 #define VENDORFILTER                    0       // set to 0 if you want to scan all devices, not filtering smartphone OUIs
-#define BLECOUNTER                      0       // set to 0 if you do not want to install the BLE sniffer
+#define BLECOUNTER                      1       // set to 0 if you do not want to install the BLE sniffer
```

Comment:
BLEcounter is intentional default off, because it has massive impact on wifi sniffer.

Reply:
done